Configuration/Base.xcconfig

```diff
@@ -38,7 +38,8 @@ WARNING_CFLAGS = -Wmismatched-tags -Wunused-private-field;
 
 OTHER_CPLUSPLUSFLAGS = $(OTHER_CPLUSPLUSFLAGS) -isystem core/include -IRealm/ObjectStore;
 
-CODE_SIGN_IDENTITY[sdk=iphone*] = iPhone Developer;
+CODE_SIGNING_REQUIRED = NO;
+CODE_SIGN_IDENTITY[sdk=*] =;
```

Comment:
if I remove `CODE_SIGN_IDENTITY` completely, iPhone and OS X build fine but watch and TV builds fail. I assume this is an artifact of them hacking in support for the latter two platforms?